amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -2720,6 +2719,58 @@ VX_API_ENTRY vx_node VX_API_CALL vxExtRppTensorAddTensor(vx_graph graph, vx_tens
     return node;
 }
 
+VX_API_ENTRY vx_node VX_API_CALL vxExtRppNormalize(vx_graph graph, vx_tensor pSrc, vx_tensor pSrcDims, vx_tensor pDst, vx_tensor pDstDims,
+                                                   vx_scalar axisMask, vx_scalar mean, vx_scalar stdDev, vx_scalar scale,
+                                                   vx_scalar shift, vx_scalar epsilon, vx_scalar ddof, vx_uint32 numOfDims) {
```

Comment:
What is the numOfDims passed here and why is it used?
Can we not infer this from pSrc?